.code-samples.meilisearch.yaml

```
@@ -175,7 +175,13 @@ update_settings_1: |-
         "synonyms": {
             "wolverine": ["xmen", "logan"],
             "logan": ["wolverine"]
-        }
+        },
+        "typoTolerance": {
+            "minWordSizeForTypos": {
+                "oneTypo": 8,
+                "twoTypos": 10
+        },
```

```suggestion
                "twoTypos": 10
                },
        },
```
I think there is a brace missing

---

I just notice that:
```
        "disableOnAttributes": ["title"]
```
Should be on the `typoTolerance` setting

---

You're right. I'll add it after line 185, thanks!